examples/tutorials/ItWaspsOnMyMachine/README.md

```diff
@@ -0,0 +1,13 @@
+# It Wasps On My Machine - an app to retrieve and store excuses for all your dev needs
+
+![Sponge Bob picks an excuse for today's stand-up](../../../web/static/img/sponge-bob-excuses.jpg)
+
+It Wasps On My Machine is a simple full stack web app, that pulls dev excuses from [
+devexcuses-api Github project](https://github.com/michelegera/devexcuses-api) and enables you to save the ones you liked (and your boss don't).
+
+## Installation
+
+Clone the repo, open the terminal in the containing folder, and run:
+
+    npm install axios
+    wasp start
```

Comment:
I think you need to add the `wasp db migrate-dev` step in here.

Reply:
Good catch!